src/Features/LanguageServer/Protocol/Features/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.cs

```diff
@@ -103,33 +88,6 @@ private void OnProjectAnalyzerReferenceChanged(object? sender, ProjectAnalyzerRe
             ClearAllDiagnostics(stateSets, project.Id);
         }
 
-        public void Shutdown()
```

Comment:
Shutdown was called when solution crawler was being shut down, which was happening when we close VS.  no need to do work here.